### ISSUE TITLE
Add interruption Chart

### ISSUE THREAD

**Issue by jaedoucette** [rifflearning/zenhub-platform-project#207]
As a PM, I want to develop and release metrics as quickly as possible, so that we can add value to our product and get customer feedback. 

As a PM, I have a December 2019 delivery for a client (GT) and would like to give them as many new metrics as possible, so that they feel like they are getting more value and that we are able to have direct customer feedback and delivery. 

Mockup of metrics layout is here: https://balsamiq-wireframes.appspot.com/?state={%22action%22:%22open%22,%22ids%22:[%221mGW0wxD7_7hzQ45EA6AYL2WDp06tRq8H%22],%22userId%22:%22115623783603948978633%22}

*** Acceptance Criteria ***

- [x] Two new Interruption metrics that use the same display as the "Influence" metrics, as described below. 
- [x] The back of the "who interrupted you" card should read "This metric shows how many times you were interrupted. Interruptions indicate dominance over the other speaker, and are not common on high-functioning teams.".
- [x] The back of the "who you interrupted card" should read "This metric shows how many times you interrupted another person. Interruptions indicate dominance over the other speaker, and are not common on high-functioning teams.".
  

** Background **
User want to see an interruption chart, showing who interrupted whom during a meeting. This story is about creating a prototype design for such a chart. 

This chart should look very similar to the existing influence charts, showing who interrupted you, and who you interrupted.

An interruption event is identical to an influence event, except that in the data is computed with a slightly different algorithm. An influence event is defined (in https://github.com/rifflearning/riff-rtc/blob/develop/src/redux/actions/dashboard.js) as:

- One person started speaking within 3 seconds of the end of someone else speaking.

An interruption is defined as:
- One person started speaking while someone else was speaking (i.e. their utterance starts while someone else is speaking).
- The overlap in speaking is > 1 second ( to account for the possibility that our detection of the end of someone's speaking was wrong).
- The person who starting speaking while someone else was speaking (the interrupter) continues to speak for at least 5 seconds, and continues to speak after the other person has stopped speaking.

The interruption chart should be identical to the existing influence chart, but with the caption "Who You Interrupted" and "Who Interrupted You" instead of "Who You Influenced" and "Who Influenced You". 

The feature should be written so that it is easy to configure the cutoffs (> 1 second, > 5 seconds), so that we can easily tune this during testing.

After the feature is implemented, we should test it during many Riff meetings.

**Comment by adonahue** [comment]
@jaedoucette - any guess on an estimate for this work?

**Comment by jaedoucette** [comment]
@adonahue  My guess is a 2 or 3. This could be done quickly by reusing the existing code for generating influence charts.

**Comment by adonahue** [comment]
And assuming this is separate from the technical work outlined in the spike report? Any estimate guess on that?

**Comment by adonahue** [comment]
@jaedoucette - would you mind taking a look at what I've written for the back of the metrics for this story and LMK if you think it should be modified?

**Comment by jaedoucette** [comment]
@adonahue Yep, I'm assuming this is separate from the technical work outlined in the spike report on Zenhub #121, in my estimate of 2 or 3 points.


Not sure about the "Interruptions indicate dominance over the other speaker, and are not common on high-functioning teams.". There are different kinds of interruptions. The kind we are measuring here are best described as "talking over someone who wasn't finished". I'm not sure that's really a desirable behavior, although maybe it is in some contexts.

Otherwise, this looks good, thanks for cleaning it up!

**Comment by adonahue** [comment]
@jaedoucette - Yeah, that description didn't feel quite right, but the purpose of the dashboard is to help people understand and change their behavior. I think this type of interruption (which doesn't need to be distinguished from the others yet b/c we don't have them) is one of the clearest examples of undesirable behavior, and it's worth saying so. But, I would prefer to ground it in something that's based on the literature. 

For example, just saying "this is how much you interrupted someone" by itself doesn't add much value as an explanation, of give people a reason not to do it. Saying what's intuitive about why it's bad (it's rude and disrespectful, people don't like it) feels too much like manners nanny. I'm trying to find language that links it to something concrete, like - teams with high rate of interruption have less less trust and effective communication. Maybe not that exactly, but in that vein. 

LMK if that makes sense, and if you have any other ideas about what might work.

**Comment by jaedoucette** [comment]
@adonahue I did some reading. It looks like this is very culture-dependent. Even within the USA there are two cultural clusters.

This is what I want to say, but it's probably too long:

"Frequent interruptions can be interpreted differently depending on the group you are speaking to. When interruptions stick to the same topic and the group is highly engaged, interruptions are likely to be perceived as positive. When interruptions abruptly change the subject, or occur in a slower-paced discussion, they are likely to be perceived as rude or disrespectful. If your team is having a lot of interruptions, consider checking in to make sure you are on the same page about what they mean."

**Comment by adonahue** [comment]
I like this quite a lot @jaedoucette, thank you! Maybe it can be trimmed a little, but all of it seems pretty relevant. And just to clarify, all of what you've written applies to the Type 1 interruptions that we're measuring, yeah?

**Comment by jaedoucette** [comment]
@adonahue Yep, it applies to the Type 1 interruptions. It probably also applies to the Type 2 kind, but we're deliberately excluding those for now. Adding them later should be a 10 minute change though.

**Comment by brecriffs** [comment]
PR posted and attached to #154

**Comment by adonahue** [comment]
Reviewed this story in Chrome, and accepted in terms of metrics display. More testing on how the metrics get generated would be useful. Text on the back of the card is different than the story criteria but makes sense given how the graphic evolved.

Card back text is still smaller than desired, and it looks like there is still whitespace to "use". Spoke to @brecriffs about the options for this and accepting story as-is.
